src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -262,6 +275,10 @@ if(BUILD_SHARED_LIBS)
         target_include_directories(${TARGET_NAME}_obj SYSTEM PUBLIC $<TARGET_PROPERTY:mlas,INCLUDE_DIRECTORIES>)
     endif()
 
+    if(ENABLE_SHL_FOR_CPU)
+        target_include_directories(${TARGET_NAME}_obj SYSTEM PUBLIC $<TARGET_PROPERTY:shl,INCLUDE_DIRECTORIES>)
```

Comment:
```suggestion
        target_include_directories(${TARGET_NAME}_obj SYSTEM PUBLIC $<TARGET_PROPERTY:shl,INTERFACE_INCLUDE_DIRECTORIES>)
```

Reply:
Applied in https://github.com/openvinotoolkit/openvino/pull/23901/commits/2dc9b522ce35211c92593e2a112a21ca8fed359e. Thanks!